src/platforms/flutter/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -212,7 +212,7 @@ SentryAssetBundle(enableStructuredDataTracing: false)
 
 ### File I/O Instrumentation
 
-The Sentry-specific file I/O implementation starts a span out of the active span, bound to the scope for each file I/O operation. The SDK sets the span `operation` to `file.copy`, `file.write`, `file.delete`, `file.open`, `file.read` or `file.rename`, and `description` to `filename` (for example, `file.txt`).
+The Sentry-specific file I/O implementation starts a span out of the active transaction, bound to the scope for each file I/O operation. The SDK sets the span `operation` to `file.copy`, `file.write`, `file.delete`, `file.open`, `file.read` or `file.rename`, and `description` to `filename` (for example, `file.txt`).
```

Comment:
```suggestion
The Sentry-specific file I/O instrumentation starts a span from an active transaction that's bound to the scope of each file I/O operation. The SDK sets the span `operation` to `file.copy`, `file.write`, `file.delete`, `file.open`, `file.read` or `file.rename` and the `description` to `filename` (for example, `file.txt`).
```

Reply:
This text has not changed, it was just a typo from `implementation` to `instrumentation`.
We use the "out of the active transaction" in multiple places, I suggest doing it in a separate PR.